dev-docs/modules/userId.md

```diff
@@ -594,6 +594,11 @@ Please reach out to [FabrickIntegrations@team.neustar](mailto:FabrickIntegration
 | params.m | | String | This is a mobile advertising ID (IDFA/AAID) used to link a user to their Fabrick ID. | |
 | params.ia | | String | This is an identifier for advertising (IFA) used to link a user to their Fabrick ID. | |
 | params.iv | | String | This is an identifier for vendors (IFV) used to link a user to their Fabrick ID. | |
+| params.1pd | | String | This is the 1st party user ID (e.g. a Customer ID/CUSTID) - Note: Requires separate delivery of identity log files keyed off the 1st party user ID to establish an identity sync. | |
```

Comment:
Could you please update this to:
"This is the 1st party user ID (e.g. a Customer ID/CUSTID). Note: This requires separate delivery of identity log files keyed off the 1st party user ID to establish an identity sync."

Reply:
Thank you for your input. I have submitted the update accordingly.